Client/Frontend/Home/JumpBackIn/SyncedTabCell.swift

```diff
@@ -0,0 +1,333 @@
+        imageView.clipsToBounds = true
+        imageView.layer.masksToBounds = true
+        imageView.layer.cornerRadius = UX.generalCornerRadius
+        imageView.backgroundColor = .clear
+        imageView.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.heroImage
+    }
+
+    private let itemTitle: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .subheadline)
+        label.numberOfLines = 2
+        label.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.itemTitle
+    }
+
+    // Contains the faviconImage and descriptionLabel
+    private var descriptionContainer: UIStackView = .build { stackView in
+        stackView.backgroundColor = .clear
+        stackView.spacing = 8
+        stackView.axis = .horizontal
+        stackView.alignment = .leading
+        stackView.distribution = .fillProportionally
+    }
+
+    let syncedDeviceImage: UIImageView = .build { imageView in
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        imageView.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.favIconImage
+    }
+
+    private let descriptionLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .caption1)
+        label.textColor = .label
+        label.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.descriptionLabel
+    }
+
+    // Used as a fallback if hero image isn't set
+    let fallbackFaviconImage: UIImageView = .build { imageView in
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        imageView.backgroundColor = UIColor.clear
+        imageView.layer.cornerRadius = TopSiteItemCell.UX.iconCornerRadius
+        imageView.layer.masksToBounds = true
+        imageView.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.fallbackFavIconImage
+    }
+
+    private var fallbackFaviconBackground: UIView = .build { view in
+        view.layer.cornerRadius = TopSiteItemCell.UX.cellCornerRadius
+        view.layer.borderWidth = TopSiteItemCell.UX.borderWidth
+    }
+
+    // Contains the hero image and fallback favicons
+    private var imageContainer: UIView = .build { view in
+        view.backgroundColor = .clear
+    }
+
+    private var syncedTabTapTargetView: UIView = .build { view in
+        view.backgroundColor = .clear
+    }
+
+    // MARK: - Variables
+    var notificationCenter: NotificationCenter = NotificationCenter.default
+
+    // MARK: - Inits
+
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+
+        isAccessibilityElement = true
+        accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.itemCell
```

Comment:
I think there's a story to improve upon that right? But I'm wondering what will be the experience in the meantime for VoiceOver?

Reply:
Yes, there is a story for improving the accessibility (#11284). Currently the experience is not optimal: voice over reads the cells content but not the show all button and the user would have to tap the correct area to open the synced tab.